### PR TITLE
Add gpgme and libassuan

### PR DESCRIPTION
**Pull request by mfojtik**
@stevekuznetsov last piece of puzzle :-)

### CODE REVIEW

File: oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml [hunk 1]
```diff
@@ -113,4 +117,4 @@
   when: origin_ci_installed_npm_version.stdout | version_compare('3.7.3', '<')
 
 - name: document all the installed packages for the build logs
-  command: '{{ ansible_pkg_mgr }} list installed'
\ No newline at end of file
+  command: '{{ ansible_pkg_mgr }} list installed'
```
Comment (stevekuznetsov) [inline oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml]: pls no

File: oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml [hunk 2]
```diff
@@ -67,6 +67,10 @@
    - xorg-x11-utils           # Origin Web Console tests
    - Xvfb                     # Origin Web Console tests
    - yum-utils                #
+   - gpgme                    # Image Signature verification dep (containers/image)
```
Comment (stevekuznetsov) [inline oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml]: alphabetize please

File: oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml (hunk 1 above)
Comment (mfojtik) [inline oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml]: hups, my vim config is always messing these up. sorry

File: oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml (hunk 2 above)
Comment (mfojtik) [inline oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml]: again, i will :sort entire list which will make the diff little bigger.

**Comment by stevekuznetsov** [comment]
@mfojtik realized we never merged this, but shouldn't we be seeing something fail with it missing? Why aren't we?

**Comment by mfojtik** [comment]
The origin PR is still waiting for the rebase to land...

**Comment by stevekuznetsov** [comment]
ack

**Pull request merged**